mscore/inspector/inspector_ambitus.ui

```diff
@@ -910,21 +894,5 @@
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>hasLine</sender>
```

Comment:
this `hasLine` is a duplicate of the above, but with different signal and receiver